commcare-cloud/commcare_cloud/commcare_cloud.py

```diff
@@ -516,6 +510,36 @@ def run_fab(args, unknown_args):
             puts(colored.red(u"✗ Fab failed with status code {}".format(exit_code)))
 
 
+class Ssh(object):
+    command = 'ssh'
+    help = "Connect to a remote host with ssh"
+
+    @classmethod
+    def make_parser(cls, parser):
+        cls.parser = parser
+        parser.add_argument("group",
+            help="Server group: postgresql, proxy, webworkers, ... The server "
+                 "group may be prefixed with 'username@' to login as a specific "
+                 "user and may be terminated with ':<n>' to choose one of "
+                 "multiple servers if there is more than one in the group. "
+                 "For example: webworkers:0 will pick the first webworker.")
+
+    @classmethod
+    def run(cls, args, ssh_args):
+        def exit(message):
+            cls.parser.error("\n" + message)
+        address = get_server_address(args.environment, args.group, exit)
+        cmd_parts = [cls.command, address] + ssh_args
```

Comment:
Ha! Clever. Maybe too clever? Since `command` here means "the name of the subcommand of `commcare-cloud`", maybe hijacking it to also mean "the name of the executable to pass these args to" is not worth the DRYness. Also not blocking, but I'd personally go with

```
command = 'ssh'
ssh_executable = 'ssh'
...
command = 'mosh'
ssh_executable = 'mosh'
```

(and then use `cls.ssh_executable` here and a few lines below) over repurposing `command` to mean both. Super-nit.

Reply:
Going to call a YAGNI here. We can add that if we ever override this class for a command that has a different executable name than whatever `command` value we pick.